Exam_11.js

```diff
@@ -0,0 +1,174 @@
+// Bài 1
+// Viết 1 hàm trả về 1 array, nhận vào 2 tham số, tham số thứ 1 là ký tự cần lặp, tham số thứ 2 là số lần lặp
+// vd: đầu vào ('ahihi', 3)
+// Thì Kết quả là:
+// ['ahihi', 'ahihi', 'ahihi']
+// Yêu cầu, viết ít nhất 2 cách khác nhau
+//method 1
+const repeatingCharacters = (a = '', n) => {
```

Comment:
```suggestion
const repeatCharacters = (a = '', n) => {
```
`a` là cái gì?
`n` là cái gì?

Viết rõ nghĩa vào